db_management.py

```diff
@@ -1,31 +1,52 @@
-import sqlite3
+from sqlalchemy import create_engine, Column, Integer, String
+from sqlalchemy.orm import sessionmaker
+from sqlalchemy.ext.declarative import declarative_base
 
 
-def new_user_creation(username_text, pass_text):
-    conn = sqlite3.connect('data_base.db')
-    curs = conn.cursor()
-    params = (username_text, pass_text)
-    curs.execute("INSERT INTO users VALUES (NULL, ?, ?)", params)
-    conn.commit()
-    conn.close()
+engine = create_engine('sqlite:///data_base.db')
+conn = engine.connect()
+Session = sessionmaker(bind=engine)
+session = Session()
```

Comment:
Not sure I understand what's the point of the Session variable here. Can't you write something like:
```suggestion
session = sessionmaker(bind=engine)
```
Or anything else?

Another point: `sessionmaker(bind=engine)` returns an object, right? how come you call a constructor on this object? Are you sure this is how it's used?

Reply:
as far as syntax goes, yes - that is how i saw it being used.
dropped "session = Session"